hack/preview-template.env

```diff
@@ -4,13 +4,15 @@
 ## Example value: origin
 export MY_GIT_FORK_REMOTE=
 
-# Optional
-
 ## HAS enable github integration
 ### Your GITHUB organization where to manage repositories by HAS
 export MY_GITHUB_ORG=
 ### Personal API token with repo and delete_repo permission
 export MY_GITHUB_TOKEN=
+
+# Optional
+
+## HAS enable github integration
```

Comment:
Was this line copied accidentally?

Reply:
nope, it's optional part for `HAS enable github integration`, so it has both optional and required